lib/api-base/fastApiContainer.ts

```diff
@@ -109,16 +112,58 @@ export class FastApiContainer extends Construct {
                 environment,
                 identifier: props.apiName,
                 instanceType: taskConfig.instanceType,
-                internetFacing: config.restApiConfig.internetFacing,
+                internetFacing: false,
                 loadBalancerConfig: taskConfig.loadBalancerConfig,
             },
             securityGroup,
             vpc,
+            addNlb: true
+        });
```

Comment:
Reading through the resources, it looks like ECSCluster->alb (apiCluster.alb) will ultimately be the load balancer that serves as the direct (unauthenticated) entrypoint to the EC2 instances that are running the ECS services for the Rest API. Is that correct?

If so, can we also add the load balancer (alb) as an externally accessible property on this construct (add to the "public readonly" entries at the top)? this would help us to stick the load balancer address into the models api and rag stacks (most likely as an environment variable) so that we can still use the endpoint for any model operations (and that would have to bubble up to stages.ts for that to get passed into those stacks)

Reply:
I just realized I'm using a SSM parameter to get the lisa serve address anyways. if that's still there then we don't need to change anything

Reply:
I don't think I changed anything there but I added it to the exposed readonly props as well :D